package.json

```diff
@@ -13,15 +13,17 @@
     "blessed": "^0.1.81",
     "blessed-contrib": "^4.7.5",
     "chalk": "^1.1.3",
+    "git-user-name": "^1.2.0",
+    "gitlog": "^2.4.0",
+    "is-git": "0.0.1",
     "parrotsay-api": "^0.1.1",
+    "resolve-dir": "^1.0.0",
     "sign-bunny": "^1.0.0",
     "scraperjs": "^1.2.0",
+    "subdirs": "^1.0.1",
     "twit": "^2.2.5",
     "weather-js": "^2.0.0"
   },
-  "peerDependencies": {
```

Comment:
@notwaldorf Wow! You actually reviewed this at #jsconfeu ! :tada: Thx. I'll try to have a look tonight.